### DISCUSSION TITLE
copilot

### DISCUSSION THREAD

**Discussion by DevLion1** [community/community#56907] [GitHub Education]
Hi everyone,

Where in the student pack do I found copilot?
I signup for the free trial but the free trial is coming to an end. 
as a student copilot help me a lot on learning of c# and would like to continue  using it as part of my education for the next two years coding Journey.
Any help will be appreciated

**Comment by samus-aran** [comment]
Hey @DevLion1  👋

We've updated our application and re-verification process to ensure that GitHub and our partners can offer exceptional, free benefits to students worldwide during their academic journey.

With this update, when you submit your application, you'll have your academic affiliation verified and gain access to Global Campus right away. Your educational benefits will arrive within 72 hours of this verification. However, during busy periods like the start of a new semester, it may take a bit longer.

You’ll get an email notification once your academic benefits are ready, or you can check your status [here](https://education.github.com/discount_requests/pack_application). Meanwhile, as a verified [GitHub Global Campus](https://education.github.com/globalcampus/student) student, you can explore the platform, connect with a local [Campus Expert](https://education.github.com/students/experts), attend events hosted by student leaders and GitHub Education, discover and contribute to student-created repositories from [GitHub Community Exchange](https://docs.github.com/en/education/explore-the-benefits-of-teaching-and-learning-with-github-education/github-global-campus-for-students/about-github-community-exchange) and much more!

If you are a verified student please retry logging in to your IDE plugin, and Copilot should work as expected.

If you still haven't received your Global Campus benefits please feel free to re-open and we'll be happy to assist!
